numpy/core/memmap.py

```diff
@@ -253,6 +254,7 @@ def __new__(subtype, filename, dtype=uint8, mode='r+', offset=0,
                 fid.seek(bytes - 1, 0)
                 fid.write(b'\0')
                 fid.flush()
+                fsync(fid.fileno())
```

Comment:
What's the motivation for this change? Is it an improvement on other platforms too?

Reply:
Maybe other platforms do synchronization inside flush(). But it is not obligatory. Without fsync() the next call of mmap() might fail. On OpenVMS it fails.

Reply:
See point 9.3 in the [openvms faq](http://www.hoffmanlabs.org/vmsfaq/vmsfaq_013.html). Although that seems like an anti-feature, I would expect [`fflush`](https://linux.die.net/man/3/fflush) to be enough.